frontend/src/redux/ts-types/popupLocation.tsx

```diff
@@ -1,3 +1,18 @@
+export type Comment = {
+  author: string;
+  text: string;
+  likes: string[];
+  dislikes: string[];
+  createdAt: {
+    type: Date;
+    default: null;
+  };
+  updatedAt: {
+    type: Date;
+    default: null;
+  };
```

Comment:
should be just:
createdAt: Date;
updatedAt: Date;

this is not a mongo schema!